pkg/controllers/infra-services-controller.go

```diff
@@ -41,10 +42,42 @@ func (s *InfrastructureServicesManager) Dependencies() []string {
 
 func (s *InfrastructureServicesManager) Run(ctx context.Context, ready chan<- struct{}, stopped chan<- struct{}) error {
 	defer close(ready)
+
+	if err := ApplyDefaultRBACs(s.cfg); err != nil {
+		klog.Errorf("%s unable to apply default RBACs: %v", s.Name(), err)
+		return err
+	}
 	// TO-DO add readiness check
 	if err := components.StartComponents(s.cfg); err != nil {
 		return err
 	}
 	klog.Infof("%s launched ocp componets", s.Name())
 	return ctx.Err()
 }
+
+func ApplyDefaultRBACs(cfg *config.MicroshiftConfig) error {
```

Comment:
Does this function need to be public if it's only called from line 46 in this file?

Reply:
this function can be private then. I'll address this.